pkg/user/manager.go

```diff
@@ -6,16 +6,31 @@ import (
 	apitypes "k8s.io/apimachinery/pkg/types"
 )
 
+type TokenInput struct {
```

Comment:
Can we move this to `pkg/auth/providers/common/usermanager.go`? This file is just for the interface as there used to be cyclical imports

Reply:
Tried that but it still creates a cyclic import
package github.com/rancher/rancher
	imports github.com/rancher/rancher/pkg/data/management
	imports github.com/rancher/rancher/pkg/agent/clean
	imports github.com/rancher/rancher/pkg/controllers/management/auth
	imports github.com/rancher/rancher/pkg/auth/providerrefresh
	imports github.com/rancher/rancher/pkg/auth/providers
	imports github.com/rancher/rancher/pkg/auth/api/secrets
	imports github.com/rancher/rancher/pkg/auth/providers/common
	imports github.com/rancher/rancher/pkg/auth/tokens
	imports github.com/rancher/rancher/pkg/types/config
	imports github.com/rancher/rancher/pkg/user
	imports github.com/rancher/rancher/pkg/auth/providers/common: import cycle not allowed